src/code42cli/cmds/devices.py

```diff
@@ -12,6 +13,8 @@
 from code42cli.click_ext.groups import OrderedGroup
 from code42cli.click_ext.options import incompatible_with
 from code42cli.click_ext.types import MagicDate
+from code42cli.cmds.legal_hold import _get_all_active_matters
```

Comment:
methods staring with `_` represent private methods, so if these methods are needed feel free to make them public.

Reply:
I'll resolve this by moving towards Tim's suggestion below